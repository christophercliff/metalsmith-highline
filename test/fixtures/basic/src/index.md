This is a paragraph [with a valid link](https://github.com/christophercliff/heroku-stripe-checkout/blob/ff1838733c5e1c99b4aa696e4d95b014177839b0/lib/start.js#L5).

[A bogus source](https://github.com/christophercliff/heroku-stripe-checkout/blob/ff1838733c5e1c99b4aa696e4d95b014177839b0/lib/start.js)

[Single line source](https://github.com/christophercliff/heroku-stripe-checkout/blob/ff1838733c5e1c99b4aa696e4d95b014177839b0/lib/start.js#L5)

[Multiple line Source](https://github.com/christophercliff/heroku-stripe-checkout/blob/ff1838733c5e1c99b4aa696e4d95b014177839b0/lib/start.js#L8-L10)

This is just another paragraph.
