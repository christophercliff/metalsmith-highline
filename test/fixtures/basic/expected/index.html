<p>This is a paragraph <a href="https://github.com/christophercliff/heroku-stripe-checkout/blob/ff1838733c5e1c99b4aa696e4d95b014177839b0/lib/start.js#L5">with a valid link</a>.</p>
<p><a href="https://github.com/christophercliff/heroku-stripe-checkout/blob/ff1838733c5e1c99b4aa696e4d95b014177839b0/lib/start.js">A bogus source</a></p>
<pre><code class="lang-js">process.on(&#39;uncaughtException&#39;, stop)
</code></pre>
<pre><code class="lang-js">server.start(function (err) {
    if (err) throw err
})
</code></pre>
<p>This is just another paragraph.</p>
